backend/app/services/ml_engine.py

```
@@ -0,0 +1,418 @@
+    history = state.get("patient_history", "").strip()
+
+    clinical_data = []
+    if query:
+        clinical_data.append(f"Жалобы/Симптомы:{query}")
+    if history:
+        clinical_data.append(f"История болезни (Анамнез):{history}")
+    if not clinical_data:
+        clinical_data.append("Жалобы и анамнез не предоставлены.")
+
+    fused_context = (
+            f"Данные пациента:\n" + "\n\n".join(clinical_data) +
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)